### DISCUSSION TITLE
OIDCRedirectURI use?

### DISCUSSION THREAD

**Discussion by bgcoder7** [OpenIDC/mod_auth_openidc#786] [Q&A]
Hi All --

`# OIDCRedirectURI is a vanity URL that must point to a path protected by this module but must NOT point to any content
OIDCRedirectURI https://www.example.com/example/redirect_uri`

Does this mean OIDCRedirectURI is just a dummy address? 
If yes, what is the use of this?

**Comment by zandbelt** [comment]
yes, it is an endpoint on which mod_auth_openidc is called back from the OP and performs various other functions
